Common Example code/Networking/Environment.swift

```diff
@@ -22,7 +22,7 @@ internal enum DemoServerEnvironment: APIEnvironment {
         }
     }
 
-    internal var version: Int { 65 }
+    internal var version: Int { 67 }
```

Comment:
should this be committed?

Reply:
This one have to, otherwise feature will not work 😛